src/helpers/agGrid.ts

```diff
@@ -575,7 +584,14 @@ export const executionToAGGridAdapter = (
     );
 
     const columnKeys = [...rowFields, ...columnFields];
-    const rowTotals = getRowTotals(totals, columnKeys, dimensions[0].headers, intl);
+    const rowTotals = getRowTotals(
+        totals,
+        columnKeys,
+        dimensions[0].headers,
```

Comment:
Isn't dimensions[0].headers already included in resultSpec?
If so, I would consider just passing dimensions[0], dimensions or just the whole resultSpec

Reply:
I see this is response, not request. By bad